community/modules/scheduler/schedmd-slurm-gcp-v6-controller/cleanup.tf

```diff
@@ -26,10 +41,5 @@ resource "null_resource" "cleanup_compute" {
     when    = destroy
   }
 
-  depends_on = [
-    # Depend on controller network, as a best effort to avoid
-    # subnetwork resourceInUseByAnotherResource error
-    # NOTE: Can not use nodeset subnetworks as "A static list expression is required"
-    var.subnetwork_self_link,
-  ]
+  depends_on = [null_resource.cleanup_compute_depenencies]
```

Comment:
Are we sure this does not instigate inopportune running of the cleanup script? For example, could adding a nodeset and redeploying cause cleanup to run now that cleanup is dependent on that new nodeset network.

Reply:
Tested adding & removing nodesets, the clean up  wasn't triggered